src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -6341,6 +6351,31 @@ uint32 ObjectMgr::GenerateLowGuid(HighGuid guidhigh)
     }
 }
 
+uint32 ObjectMgr::GenerateRecycledLowGuid(HighGuid guidHigh)
+{
+    switch (guidHigh)
+    {
+        case HIGHGUID_UNIT:
+        {
+            ASSERT(_hiCreatureRecycledGuid < 0x00FFFFFE && "Creature recycled guid overflow!");
+            if (_hiCreatureRecycledGuid < _hiCreatureRecycledGuidMax)
+                return _hiCreatureRecycledGuid++;
+            break;
+        }
+        case HIGHGUID_GAMEOBJECT:
+        {
+            ASSERT(_hiGoRecycledGuid < 0x00FFFFFE && "Gameobject recycled guid overflow!");
+            if (_hiGoRecycledGuid < _hiGoRecycledGuidMax)
+                return _hiGoRecycledGuid++;
+            break;
+        }
+        default: // No es manejado por el reciclador
```

Comment:
What?

Reply:
default case is not handled by the recycler, should I change the comment or just remove it ? (and thank you for noticing)

Reply:
i translated it, i guess i can merge right?

Reply:
@Winfidonarleyan feel free to merge it when you see that message :P

Reply:
Yes, need translating;D